### ISSUE TITLE
"merge from database" without asking the password every time 

### ISSUE THREAD

**Issue by aisbergde** [keepassxreboot/keepassxc#4051]
it should be easier to merge databases, the password should not be required on each sync, but only once or it should be not required at all if the password in both database files is the same.

For many years I use KeePass and I synchronize databases on each saving and there is no additional password request if both files have the same password. This works great and some entries can be changed on the desktop, others can be changed or added on Android.

KeePassXC has no synchronization build in, but only the merge feature. I can use it manually, this is fine. But there should be an easy way to merge files often. KeePass will synchronize on each save if cloud synchronization is used. KeePassXC should enable to work a similar way.

**Comment by phoerious** [comment]
I am not sure what you mean. If the database changed but the password didn't, it is simply reloaded without a prompt.

**Comment by aisbergde** [comment]
Currently when I use 
Menu > Database > Merge from database
- I need to select the database I want to merge
- I need to enter the password each time

On the other side:
I KeePass I have
Menu > File > Synchronize
now I can use "recent files" or I can select a new file
the synchronization starts and I don't need to enter the password

I want to request this feature in KeePassXC

**Comment by droidmonkey** [comment]
Duplicate of #2937
